{{ cookiecutter.project_name }}/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+#### Description
+
+<!-- Please include a summary of the change and the relevant issue(s) it
+resolves, if any (otherwise delete that line), e.g., `Fixes #123`. If the PR
+addresses more than one issue, please add multiple lines, each starting with
+'Fixes #'. Please stick to that syntax precisely, including whitespaces,
+otherwise the issue(s) may not be linked to the PR.
+
+In the summary, list any dependencies that are required for this change.
+Please use bullet points for the description. Please also briefly describe
+the relevant motivation and context briefly. For very trivial changes that are
+duly explained by the PR title, a description can be omitted. -->
+
+- Fixes #(issue number)
```

Comment:
Does this work with the list bullets? I'm not sure.

I think it should rather be:

```
Fixes #(issue number)

- 
```

Reply:
yes as long as there is a hash/pound sign it sould work, but do we want that, its way easier to just keep on listing the issues a PR fixes, like #1 #2 #3 etc etc.

Reply:
No, it's not enough. It definitely also needs the "Fixes" keyword immediately preceding the `#(issue number)`. See https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

I haven't tested whether it works if there is something before "Fixes", but generally this mecahnism is pretty fickle, so I wouldn't count on it.

What for sure also doesn't work is something like this:

```
Fixes #1, #2
Fixes #1 #2
```

Each linked issue has to be on its own line.

You can try out in a private repo if the list syntax works, if you like. But without testing it, I think we should just be on the safe side on use this:

```
Fixes #1 
Fixes #2 
```

and so on...

Reply:
hmm, lets go with the latter.